tests/test_callbacks.py

```diff
@@ -4,4 +4,4 @@ def test_when_imagej_starts(ij):
     when_imagej_starts, which injects a small piece of data into the gateway
     object. We check for that data here to make sure the callback happened.
     """
-    assert "success" == getattr(ij, "_when_imagej_starts_result", None)
+    assert True is getattr(ij, "_testing", True)
```

Comment:
This assertion will pass if the `_testing` property is unset or does not exist, because you're asking for `True` back by default in those cases... so it's not testing for the correct behavior.

Reply:
Oops, my mistake! Nice catch! You're right, the `default` here needs to be `None`. Fixed and tested in b2ce4cb21836bacc68338c6b2dc84fc879e081fd.